tox.ini

```diff
@@ -487,7 +491,7 @@ whitelist_externals =
 commands =
     mkdir -p test-results
     wandb login --relogin {env:WANDB_API_KEY}
-    standalone-cpu-py{36,37,38,39,310}: yea --strict -p wandb:mockserver-relay=true -p wandb:mockserver-relay-remote-base-url=https://api.wandb.ai --shard standalone-cpu run {posargs:--all}
-    standalone-gpu-py{36,37,38,39,310}: yea --strict -p wandb:mockserver-relay=true -p wandb:mockserver-relay-remote-base-url=https://api.wandb.ai --shard standalone-gpu run {posargs:--all}
-    standalone-tpu-py{36,37,38,39,310}: yea --strict -p wandb:mockserver-relay=true -p wandb:mockserver-relay-remote-base-url=https://api.wandb.ai --shard standalone-tpu run {posargs:--all}
-    standalone-local-py{36,37,38,39,310}: yea --strict -p wandb:mockserver-bind=0.0.0.0 -p wandb:mockserver-host=__auto__ -p wandb:mockserver-relay=true -p wandb:mockserver-relay-remote-base-url=http://localhost:5000 --shard standalone-cpu run {posargs:--all}
+    standalone-cpu-py{36,37,38,39,310}: yea --debug --strict -p wandb:mockserver-relay=true -p wandb:mockserver-relay-remote-base-url=https://api.wandb.ai --shard standalone-cpu run {posargs:--all}
```

Comment:
Wouldn't output be huge with debug mode on?

Reply:
it could be bad if the test is doing a whole lot..   so.. definitely not ideal, but workable with the current tests.. ill run one last nightly before merging.   I just wanted to see the output until we got a chance to add some top level yea functionality to display profiling summaries and coverage summaries.